docs/pages/setup/guides/fluentd.mdx

```diff
@@ -44,11 +44,21 @@ from Teleport's events API, and forwards them to Fluentd.
   $ tar -zxvf teleport-event-handler-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
   ```
   </TabItem>
+
   <TabItem label="Docker">
   ```code
   $ docker pull quay.io/gravitational/teleport-plugin-event-handler:(=teleport.plugin.version=)
   ```
   </TabItem>
+
+  <TabItem label="Helm">
+  ```code
+  $ helm repo add teleport https://charts.releases.teleport.dev/
+  ```
+
+  See the Helm Chart example under [Step 6/6. Start the event handler plugin](#step-66-start-the-event-handler-plugin) to see how to create `teleport-plugin-event-handler-values.yaml`!
```

Comment:
Would a Helm user still need to create a Teleport user/role for reading audit events, enable impersonation for the `teleport-event-handler` user/role, and export credentials? If so, I think we should delete this sentence and make sure each of steps 2-6 includes Helm instructions.

For example, we can rename Step 2 to "Configure the plugin", remove the instructions below the `Tabs` box, and copy those instructions into the `TabItem`s for "Self-Hosted" and "Teleport Cloud". We could then add a third tab for "Helm" that provides instructions on creating a values file. 

Currently, the values file instructions are in Step 6, which is why we need to jump around the doc.